### PR TITLE
British English: colour

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

File: src/4.6/en/textui.xml [hunk 1]
```diff
@@ -159,7 +159,7 @@ Test Execution Options:
   --no-globals-backup       Do not backup and restore $GLOBALS for each test.
   --static-backup           Backup and restore static attributes for each test.
 
-  --colors=<flag>           Use colors in output ("never", "auto" or "always").
+  --colors=<flag>           Use colours in output ("never", "auto" or "always").
```
Comment (henriquemoody) [inline src/4.6/en/textui.xml]: This is the output from `phpunit --help`, unless you send a pull request to [PHPUnit](https://github.com/sebastianbergmann/phpunit/) it cannot be changed.

File: src/4.6/en/textui.xml [hunk 2]
```diff
@@ -634,7 +634,7 @@ class TestCaseClass extends \PHPUnit_Framework_TestCase
         <term><literal>--colors</literal></term>
         <listitem>
           <para>
-            Use colors in output.
+            Use colours in output.
```
Comment (henriquemoody) [inline src/4.6/en/textui.xml]: This must be updated in others versions too.

**Comment by franzholz** [comment]
I have added the required pull request.

https://github.com/sebastianbergmann/phpunit/pull/1572

**Comment by whatthejeff** [comment]
I discussed this with @sebastianbergmann on gitter. We decided to stick with the current spelling since American English is more ubiquitous in the PHP world.

**Pull request closed without merging**